x/wasm/keeper/query_plugins.go

```diff
@@ -269,8 +272,32 @@ func IBCQuerier(wasm contractMetaDataSource, channelKeeper types.ChannelKeeper)
 }
 
 func StargateQuerier(queryRouter GRPCQueryRouter) func(ctx sdk.Context, request *wasmvmtypes.StargateQuery) ([]byte, error) {
```

Comment:
I noticed that https://github.com/terra-money/classic-core/pull/738/files#diff-f362187ed46ff14698b8df7a092e012f879ae79989b34e4b2c1531af8730f29f 

has tests, can we port some of them over or add our own custom tests?

Reply:
added test cases!